test/dotnet-publish.Tests/GivenDotnetPublishPublishesProjects.cs

```diff
@@ -77,6 +74,46 @@ public void ItPublishesARunnableSelfContainedApp()
                      .And.HaveStdOutContaining("Hello World");
         }
 
+        [Theory]
+        [InlineData("1.1.9-servicing-001731-00")]
```

Comment:
I think we need to figure out a way not to hard-code this in the test.  Some options would be to look at what version is available in the Shared/Microsoft.NETCore.App folder in the .NET Core SDK that's being used, to read the version from Microsoft.DotNet.Cli.DependencyVersions.props, or use the MSBuild property to write a text file which would be included as an embedded resource in the test assembly.